apps/api/src/coscrad-cli/domain-dump.cli-command.ts

```diff
@@ -18,18 +19,18 @@ export class DomainDumpCliCommand extends CommandRunner {
         this.dataExporter = new DataExporter(repositoryProvider);
     }
 
-    async run(_passedParams: string[], options?: Record<string, any>): Promise<void> {
+    async run(_passedParams: string[], options?: { filepath: string }): Promise<void> {
         const NUMBER_OF_SPACES_TO_INDENT = 4;
 
-        const result = await this.dataExporter.fetchSnapshot();
+        const domainSnapshot = await this.dataExporter.fetchSnapshot();
+
+        const snapshotInDatabaseFormat = convertInMemorySnapshotToDatabaseFormat(
```

Comment:
We should add a story to expose a cli option to export the domain or view format instead.

Reply:
[Here](https://www.pivotaltracker.com/story/show/185103607) it is.